pkg/algorithm/cstorpoolselect/v1alpha1/select_test.go

```diff
@@ -157,3 +165,152 @@ func TestNewSelection(t *testing.T) {
 		})
 	}
 }
+
+func TestAntiAffinityFilter(t *testing.T) {
+	tests := map[string]struct {
+		CVRUid, poolUids, expectedUids []string
+		labelSelector                  string
+		expectError                    bool
+	}{
+		"Test 1":  {[]string{}, []string{"uid 1", "uid 2", "uid 3"}, []string{}, "label3", true},
```

Comment:
do we make use of CVRUid in source code?

Reply:
nope